release-notes/major9/09.26.0/valtimo-frontend-libraries.md

```diff
@@ -31,9 +31,9 @@ The following features were added:
 
 The following bugs were fixed:
 
-* **Bug1**
+* **Size of collection in Dossier list**
```

Comment:
Please use the english name, so `case` instead of `Dossier` (also no capitalization please!). Same goes for the below below.